beacon_node/beacon_processor/src/scheduler/work_reprocessing_queue.rs

```diff
@@ -756,13 +757,17 @@ impl<S: SlotClock> ReprocessQueue<S> {
             }
             InboundEvent::Msg(DelayColumnReconstruction(request)) => {
                 let mut reconstruction_delay = QUEUED_RECONSTRUCTION_DELAY;
+                let slot_duration = self.slot_clock.slot_duration().as_millis() as u64;
+                let reconstruction_deadline_millis =
+                    (slot_duration * RECONSTRUCTION_DEADLINE.0) / RECONSTRUCTION_DEADLINE.1;
+                let reconstruction_deadline = Duration::from_millis(reconstruction_deadline_millis);
                 if let Some(seconds_from_current_slot) =
                     self.slot_clock.seconds_from_current_slot_start()
```

Comment:
This function only gives us granuarity to the seconds I believe

https://github.com/sigp/lighthouse/blob/522bd9e9c6ac167f2231525e937c9ebbcb86cf6e/common/slot_clock/src/lib.rs#L108-L110

So on a network like gnosis (5s slot times), reconstruction is only triggered immediately if `seconds_from_current_slot_start` is 2s